functorch/test/test_aotdispatch.py

```diff
@@ -886,6 +886,7 @@ def assert_compiler(gm: torch.fx.GraphModule, _):
     xfail('nn.functional.avg_pool1d', ''),  # Cannot call sizes() on tensor with symbolic sizes/strides
     xfail('nn.functional.avg_pool2d', ''),  # aten.avg_pool2d.default - couldn't find symbolic meta function/...
     xfail('nn.functional.avg_pool3d', ''),  # aten.avg_pool3d.default - couldn't find symbolic meta function/...
+    skip('nn.functional.batch_norm', ''),  # '0 is not tracked with proxy for <torch.fx.experimental.proxy_te..
```

Comment:
I'll try to remember to debug / remove this when https://github.com/pytorch/pytorch/pull/84246/commits/247d5db4951f312e6ac86fe59b22df13b68329d4?notification_referrer_id=NT_kwDOAPjlw7M0Mjk0NzkyNjMxOjE2MzExNzQ3 makes it to master